task/gcloud/0.3/gcloud.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: gcloud
+  labels:
```

Comment:
since we are running everything gcp, is there a way that we can test this Task?
/cc @afrittoli @bendory @jerop

Reply:
It certainly is testable -- at a minimum, `gcloud version` should succeed; `gcloud config list` could be compared to known golden output. I just looked in the `test` directory but I don't see any documentation on adding tests; is there a `README` or perhaps a sample test I can use as a guide?

Reply:
probably you can refer the following https://github.com/tektoncd/catalog/tree/main/task/black/0.2 😅. Do let me know if you need any assistance, happy to help 🙂

Reply:
Based on what I see in the `black` tests that you reference, I added a few tests -- but these are more like samples than tests, is there testing infrastructure for catalog or have I done what you're asking for?

I see other tasks have a `samples` directory, perhaps these belong in `samples`?

Reply:
Note: the `config.yaml` and `version.yaml` I added are platform and environment agnostic. I could add samples or tests of GCP interactions, but they will be platform and environment dependent.

Reply:
Aha! I just found the [guidance on e2e testing](https://github.com/tektoncd/catalog/blob/main/CONTRIBUTING.md#end-to-end-testing). Reverting to draft while I digest + implement that.